swagger/coordinator.yml

```diff
@@ -0,0 +1,116 @@
+openapi: 3.0.1
+info:
+  title: XAIN FL Coordinator API
+  contact:
+    email: engineering@xain.io
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.0
+externalDocs:
+  description: Find out more about XAIN FL
+  url: https://docs.xain.io/
+servers:
+  - url: http://localhost:8081/
+tags:
+  - name: Coordinator
+    description: API of the coordinator service
+paths:
+  /rendez_vous:
+    get:
+      tags:
+        - Coordinator
+      responses:
+        200:
+          description: client accepted
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/RendezVousResponse"
+        404:
+          description: ""
+          content: {}
+  /heartbeat/{client_id}:
+    get:
+      tags:
+        - Coordinator
+      parameters:
+        - name: client_id
+          in: path
+          description: ID of the client
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: heartbeat confirmed
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/HeartBeatResponse"
+        404:
+          description: client unknown
+          content: {}
+  /start_training/{client_id}:
+    get:
+      tags:
+        - Coordinator
+      parameters:
+        - name: client_id
+          in: path
+          description: ID of the client
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: client is selected for training round
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/StartTrainingResponse"
+        404:
+          description: ""
+          content: {}
+components:
+  schemas:
+    RendezVousResponse:
+      type: object
+      properties:
+        id:
+          type: string
+          description: client ID
+          example: 1fa2f908-83e2-4f92-80e1-4baee0bf25a5
+        ok:
+          type: boolean
+          description: ""
```

Comment:
Whether the rendez-vous was accepted (`true`) or rejected (`false`) by the coordinator. A participant that has been rejected should not try to recontact the coordinator.